modules/basis/basis-datasource/samples/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM tomcat:8.5.38-jre8
+
+RUN rm -fR $CATALINA_HOME/webapps/ROOT
+
+COPY *.war $CATALINA_HOME/webapps/
+
+COPY *.jar $CATALINA_HOME/lib/
+
+RUN sed -i /'<Context>'/c'<Context>\n<Listener className=\"io.dirigible.tomcat.listener.NamingResourcesListener\" factoryClassName=\"io.dirigible.smart.datasource.object.factory.SmartDatasourceObjectFactory\"/>' $CATALINA_HOME/conf/context.xml
+
```

Comment:
As a next step, this has to be also integrated into the other Dockerfiles:
- https://github.com/SAP/xsk/blob/main/releng/server/Dockerfile
- https://github.com/SAP/xsk/blob/main/releng/server/Dockerfile-external-jdk
- https://github.com/SAP/xsk/blob/main/releng/sap-kyma/Dockerfile
- https://github.com/SAP/xsk/blob/main/releng/sap-kyma/Dockerfile-external-jdk
- https://github.com/SAP/xsk/blob/main/releng/sap-cf/Dockerfile
- https://github.com/SAP/xsk/blob/main/releng/sap-cf/Dockerfile-external-jdk

Reply:
done